src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/methodparampad/InputMethodParamPadWhitespaceAround.java

```diff
@@ -37,5 +37,5 @@ public void enhancedFor ()
 }
 
 @interface CronExpression1 {
-    Class<?>[] groups() default { }; // extra space
+    Class<?>[] groups() default { };
```

Comment:
`// extra space between curly braces, ok`

Reply:
This check should not be expected to validate curly braces or annotation type elements. I left comments in the input file to make this obvious.